src/lib/cimb_translator/Config.cpp

```diff
@@ -1,94 +1,38 @@
 /* This code is subject to the terms of the Mozilla Public License, v.2.0. http://mozilla.org/MPL/2.0/. */
 #include "Config.h"
+#include <cmath>
 
 namespace cimbar {
 
-bool Config::dark()
+unsigned Config::total_cells()
 {
-	return true;
+	return std::pow(cells_per_col(), 2) - std::pow(corner_padding(), 2) * 4;
```

Comment:
Most of this has been moved to the header file. The cpp is now just the "smart" helper functions.